fsharp/base/FStar_Option.fs

```diff
@@ -1,4 +1,5 @@
 module FStar_Option
+open FStar_Pervasives_Native
```

Comment:
Are these opens really needed? Just curious.

Reply:
I believe so, otherwise the `None`, `Some` etc. below would resolve to the F# primitives, rather than the ones defined in FStar_Pervasives_Native